### ISSUE TITLE
Low contrast on dropped dice

### ISSUE THREAD

**Issue by gareththegeek** [gareththegeek/rollagon#81]
The dropped dice (dice which did not contribute to the final roll) are failing accessibility checking for low contrast:

![image](https://user-images.githubusercontent.com/6231841/173054807-f3da44f2-3178-48ea-b2e3-eebebebb2770.png)

**Comment by sentinelgreg** [comment]
I'm happy to change this - Let's just make them "Color 2" = 004C48

**Comment by gareththegeek** [comment]
Done #82
